packages/apollo-language-server/src/project.ts

```diff
@@ -241,12 +294,50 @@ export class GraphQLProject {
     );
   }
 
+  async updateSchemaTag(tag: string) {
+    await this.loadingHandler.handle(
+      `Loading queries and schemas for ${this.config.name!}`,
+      (async () => {
+        this.documentSets = await resolveDocumentSets(this.config, true, tag);
+        for (const set of this.documentSets) {
+          if (!set.schema!.getQueryType()!.astNode) {
```

Comment:
lots of type overrides here, are we sure we won't hit errors?

Reply:
d0a34efa1371fdbc080a1c931027c79d3cb8a898